src/app/query/common/market-data/vendors/coincap-market-data.query.ts

```diff
@@ -9,6 +9,7 @@ import { marketDataQueryOptions } from '../market-data.query';
 const currencyNameMap: Record<CryptoCurrencies, string> = {
   BTC: 'bitcoin',
   STX: 'stacks',
+  ALEX: 'alex',
```

Comment:
```suggestion
```

We can never know all the SIP-10 tokens, so IMO we should only put the known-currencies in this list

Reply:
it is used in fetchMarketData function, you assume to create other type? or don't fetch prices for sip10 tokens?

Reply:
This doesn't makes sense bc ALEX isn't the only sip10 token. We would have to add all fungible tokens on Stacks. (USDA, DIKO, etc).